src/Parser/Banking/Mt940/Engine/Wise.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Kingsquare\Parser\Banking\Mt940\Engine;
+
+use Kingsquare\Parser\Banking\Mt940\Engine;
+
```

Comment:
`use Kingsquare\Banking\Transaction\Type;` use statement is missed.

Reply:
Yes, I forgot to use type. I added it now. Thanks your feedback